lib/utils/type.js

```diff
@@ -35,7 +35,7 @@ async function detectTypeUsingFileCommand(renditionPath) {
         }
 
     } catch (e) {
-        console.log("file command failed:", e);
+        console.log(`Warning: executing 'file' to detect file type failed with: ${e.message}. code: ${e.code} signal: ${e.signal}`);
```

Comment:
This or `console.warn` ?

Reply:
I think prefixing it with `Warning` and leaving it as `log` is fine